shaders/CMakeLists.txt

```diff
@@ -19,8 +19,10 @@ foreach(SHADER ${SHADERS})
         COMMAND ${GLSL_VALIDATOR} -V ${CMAKE_CURRENT_SOURCE_DIR}/${SHADER} -o ${SPIRV}
         DEPENDS ${SHADER}
     )
-
+    
     list(APPEND SPIRV_FILES ${SPIRV})
 endforeach()
 
-add_custom_target(inexor-shaders DEPENDS ${SPIRV_FILES})
+source_group("Shader Files" FILES ${SHADERS})
+
+add_custom_target(inexor-shaders DEPENDS ${SPIRV_FILES} SOURCES ${SHADERS})
```

Comment:
Can you add a line break?

Reply:
Already a new line at the end of file, GH doesnt render it